java/src/jmri/jmrix/loconet/LocoNetBundle_nl.properties

```diff
@@ -124,3 +124,6 @@ StatusWriting = schrijf SV{0} (poort {1} V{2})
 StatusWriteMode = schrijf SV{0} (poort {1} mode)
 AddressToolTip = Vul het LocoIO Module Laag Adres in (hex 01-7f, m.u.v. 0x50 = dec 80)
 SubAddressToolTip = Vul het LocoIO Module Hoog Adres in (hex 01-7f = dec 1-126)
+
+# Used by several different LocoNet-based tools
+ButtonCancel = Zru\u0161it
```

Comment:
Thanks for taking care of this bundle, but the value of "Stop" would be fine in Dutch (_nl). I reckon this is the Czech (_cz) text.

Reply:
Watch for an upcoming commit which reverts this modification.